.tekton/o11y-pull-request.yaml

```diff
@@ -280,7 +280,7 @@ spec:
         - name: url
           value: https://github.com/tektoncd/catalog.git
         - name: revision
-          value: main
+          value: f4708d478ee8fac6b5b68347cde087cb7c1d1b1c
```

Comment:
What is this change about?

Reply:
I created [another PR](https://github.com/redhat-appstudio/o11y/pull/209) for this. You can remove the changes here and rebase.

Reply:
EC was failing with violation `msg: Task kube-linter is used by pipeline task run-kubelint via an unpinned reference` so I have updated it to use recent commit sha from the task used over there.

Sure @yftacherzog, I'll remove the changes here